app/src/main/java/com/sopt/now/feature/search/ReqresAdapter.kt

```diff
@@ -0,0 +1,30 @@
+package com.sopt.now.feature.search
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.ListAdapter
+import com.sopt.now.core.view.ItemDiffCallback
+import com.sopt.now.databinding.ItemFriendBinding
+import com.sopt.now.domain.entity.ReqresEntity
+import com.sopt.now.feature.search.viewholder.ReqresViewHolder
+
+class ReqresAdapter : ListAdapter<ReqresEntity, ReqresViewHolder>(reqresDiffCallback) {
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ReqresViewHolder {
+        val inflater = LayoutInflater.from(parent.context)
+        val binding = ItemFriendBinding.inflate(inflater, parent, false)
+        return ReqresViewHolder(binding)
+    }
```

Comment:
[P1]
inflater를 매번 재생성하는 대신,  lazy를 활용하여 재사용하는 것은 어떨까요?
```suggestion
class ReqresAdapter(context: Context) : ListAdapter<ReqresEntity, ReqresViewHolder>(reqresDiffCallback) {
    private val inflater by lazy { LayoutInflater.from(context) }
    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ReqresViewHolder {
        val binding = ItemFriendBinding.inflate(inflater, parent, false)
        return ReqresViewHolder(binding)
    }
```